packages/front-end/styles/global.scss

```diff
@@ -1009,6 +1009,20 @@ main.main {
   td {
     vertical-align: middle;
   }
+
+  &.decision-criteria-table .dark {
```

Comment:
issue: This style is not being applied because there's no `.dark` element inside the table.

Do we need to update it to be like the rule below (line 1027)? `.dark &.decision-criteria-table {`?

Otherwise I really think these colors work much better! Thanks for the update.